lib/models/Restaurant.js

```diff
@@ -0,0 +1,48 @@
+const pool = require('../utils/pool');
+const { Review } = require('./Review');
+
+class Restaurant {
+  id;
+  name;
+  cuisine;
+  cost;
+  image;
+  website;
+
+  constructor(row) {
+    this.id = row.id;
+    this.name = row.name;
+    this.cuisine = row.cuisine;
+    this.cost = row.cost;
+    this.image = row.image;
+    this.website = row.website;
+  }
+
+  static async getAll() {
+    const { rows } = await pool.query(`
+        SELECT * FROM restaurants
+        `);
+    return rows.map((row) => new Restaurant(row));
+  }
+  static async getById(id) {
+    const { rows } = await pool.query(
+      ` SELECT * FROM restaurants 
+        WHERE id = $1
+        `,
+      [id]
+    );
+    if (!rows) return null;
```

Comment:
i think you need to to check ```if (!rows[0])``` because if there is no row with id that is passed in, i believe your response will still be an empty array which evaluates as truthy